workers/unity/Packages/com.improbable.gdk.core/Utility/SnapshotBuilder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class SnapshotBuilder
+    {
+        private readonly Dictionary<EntityId, Entity> entities = new Dictionary<EntityId, Entity>();
+
+        public int Count => entities.Count;
+
+        public void AddEntity(Entity entity)
```

Comment:
Maybe over-engineering this - but this probably wants an API for a specific entity with a specific entity id.

Given like a spawner entity where you expect it to always be id 1